graphql-api/src/queries/lof-curation-result-queries.ts

```diff
@@ -1,12 +1,21 @@
-const GNOMAD_V2_LOF_CURATION_RESULTS_INDEX = 'gnomad_v2_lof_curation_results'
+type GnomadVersion = 'ExAC' | 'v2'
+
+const GNOMAD_LOF_CURATION_RESULTS_INDICES = {
+  ExAC: 'gnomad_v2_lof_curation_results',
+  v2: 'gnomad_v2_lof_curation_results',
+}
 
 // ================================================================================================
 // Variant query
 // ================================================================================================
 
-export const fetchLofCurationResultsByVariant = async (esClient: any, variantId: any) => {
+export const fetchLofCurationResultsByVariant = async (
+  esClient: any,
+  gnomadVersion: GnomadVersion,
+  variantId: any
```

Comment:
while we're in here let's give a proper type, probably simply `string`, to `variantId`. some of the stuff I'm working on currently will also add a type for ES clients so no need to worry about that ATM.